src/components/Admin/Input.jsx

```diff
@@ -1,26 +1,26 @@
 import React from "react";
 import { MdCancel } from "react-icons/md";
 
-const Input = ({ value, setValue, clear, label, maxLength, placeholder }) => {
+const Input = ({ object, setObject, clear, label, maxLength, placeholder }) => {
   return (
     <div className="flex items-center">
-      <p className="text-lg font-extrabold mr-2 my-0">{label}</p>
+      <p className="text-lg font-extrabold mr-2 my-0">{label}:</p>
       <div className="flex items-center my-1 bg-hackathon-gray rounded-md w-full">
         <input
-          value={value}
+          value={object[label]}
           maxLength={maxLength}
           placeholder={placeholder}
           type="text"
           className="outline-none w-full font-poppins text-base py-1 bg-transparent px-2"
           onChange={(e) => {
```

Comment:
make this a function, otherwise u have to copy paste the setObject thing multiple times, the entire point was to prevent the copy paste